testing/hexer/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=hexer
+pkgver=1.4.0
+pkgrel=0
+pkgdesc="LAS and OGR hexagonal density and boundary surface generation"
+url="https://github.com/hobu/hexer"
+arch="all"
+license="LGPL2+"
+options="!check"
+depends="libgcc libstdc++ musl gdal"
+makedepends="cmake gdal-dev"
+install=""
+subpackages="$pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::https://github.com/hobu/$pkgname/archive/$pkgver.tar.gz
+        10-missing-includes.patch
+        20-backtrace-guards.patch"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+        mkdir -p "$builddir"
+        cd "$builddir"
+
+        mkdir build && cd build
+        cmake .. \
+                -DCMAKE_INSTALL_PREFIX=/usr \
+                -DCMAKE_BUILD_TYPE=Release
+        make
+}
+
+package() {
+        cd "$builddir"/build
+        make DESTDIR="$pkgdir" install
+}
+
+prepare() {
+        default_prepare
+}
```

Comment:
You can remove this `prepare()` function since this is exactly what the abuild provided `prepare()` does.